internal/cli/atlas/alerts/settings/settings.go

```diff
@@ -101,49 +116,83 @@ func (opts *ConfigOpts) newNotification() *admin.AlertsNotificationRootForGroup
 		out.NotifierId = &opts.notifierID
 	}
 
+	// write set of functions that contain code from switch cases and return error if required fields are not provided
 	switch out.GetTypeName() {
```

Comment:
Based on the type we perform different validation.
Without validation it would be almost impossible for the end user to understand what values are required for the type.